### DISCUSSION TITLE
Show home screen after an idle time

### DISCUSSION THREAD

**Discussion by adocampo** [OpenVoiceOS/ovos-buildroot#91] [Ideas]
Just tested the [video] player, which opens an embedded player and plays videos from youtube. 
Once the video/playlist is complete, a black screen with three white dots per pagination is presented and it stays like this _ad eternum_
Would be nice if after some minutes of idle time (no music/video playing), home screen is presented again without user interaction.

**Comment by JarbasAl** [comment]
this is fixed but not yet in latest image, when playback ends you should be taken to the main ocp page where you can search for more music. 

There is also a autoplay option and playlist support now, meaning that when a track ends the next one will start automatically

**Reply by adocampo** [reply]
> when playback ends you should be taken to the main ocp page where you can search for more music.

And what about when I'm idle on that OCP page for several minutes? It comes to home screen or stays there until human interaction?

**Comment by JarbasAl** [comment]
if you open it by voice/click it stays open until you do something, if it shows when music playback ends it goes to homescreen after 30 seconds

**Reply by adocampo** [reply]
I should wait for an update then, because here it stays forever in black when playback ends.

**Reply by JarbasAl** [reply]
OCP is running quite behind in the latest image, the next one will have a lot of updates in regards to common play :)

**Comment by j1nx** [comment]
We will need to double check if this is solved in the latest and just release pre-release.
